content/Joypad_Input.md

```diff
@@ -31,17 +31,17 @@ to the SNES. See SGB description for details.
 ### INT 60 - Joypad Interrupt
 
 The Joypad interrupt is requested when any of the above Input lines changes
```

Comment:
"Input lines" should be changed to "bits 0-3" to better hint that bit 4 and/or 5 need to select them.

Reply:
Fixed

Reply:
Fixed